### DISCUSSION TITLE
评论邮箱通知、回复

### DISCUSSION THREAD

**Discussion by github-wsz** [walinejs/waline#765] [Q&A] [answered]
我在部署设置面板设置了QQ邮箱相关的环境变量，重启后不成功，不知道是哪里设置不对

**Comment by lizheming** [comment, marked as answer]
可以看下服务端日志是否有具体的报错信息。另外 QQ 邮箱的密码是独立密码，不是 QQ 密码，这个需要确认下。

**Reply by github-wsz** [reply]
抱歉，日志我不知道在哪里看，qq邮箱密码我刚刚尝试了使用独立密码，还是无法收到回复评论的通知
![image](https://user-images.githubusercontent.com/45225757/149612834-141881d4-dada-43a8-b447-7d46ff2ff073.png)

**Reply by sszxc** [reply]
我尝试了 163 和 Gmail 都没有成功。也请问一下日志在哪里能查看？

**Reply by lizheming** [reply]
Vercel 后台查看

**Reply by sszxc** [reply]
您好，请问是在 function - realtime logs 下面吗，这边一直显示 “Awaiting new requests…”，不知道是不是网络问题

**Reply by sszxc** [reply]
知道我的问题了，每次 redeploy 之后服务端地址都会改变，要去博客里重新配置一下...
虽然还是没找到在哪看日志哈哈😂

**Reply by lizheming** [reply]
项目里是有一个地址是指向最新部署的版本的，不需要每次修改地址

**Reply by sszxc** [reply]
感谢！问题彻底解决
